core/playwright/externalFormCreatePub.spec.ts

```diff
@@ -70,11 +70,13 @@ test("Can create a pub from an external form", async () => {
 
 test.describe("Multivalue inputs", () => {
 	test("Can add a radio and checkbox multivalue input", async () => {
+		test.setTimeout(60_000);
```

Comment:
🙈

Reply:
This is just overriding the maximum amount of time this test can run before failing right?

Reply:
yup that's right! default is 30s